CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.30.0
+
+- Patching: location_in_specfile has been removed from metadata.
+- Synced_files mechanism has been refactored.
+- Bug with fedora-latest alias has been fixed.
```

Comment:
Past perfect sounds strange, how about:

```suggestion
- Patching: removed location_in_specfile from commit metadata.
- Refactored and extended the synced_files mechanism.
- Fixed a bug regarding the fedora-latest alias.
```

Would be nice to link the issue or the PR (if there was no issue) for the fedora-latest bug.